src/Twig/Components/MagazineInlineComponent.php

```diff
@@ -15,4 +15,5 @@ final class MagazineInlineComponent
     public bool $fullName = false;
     public bool $stretchedLink = false;
     public bool $showAvatar = false;
```

Comment:
TIL: These variables are defined in the Twig component as well with default values. I believe this `showAvatar` should be set to `true` as well, since that is the default option we also have in the GUI. And then its consistent with the `showAvatar` value as well in `UserInlineComponent.php` (which was already true).

After that, I believe we can remove a lot of unnecessary setting booleans to true, and see if that this works. Since then Twig will just use the default boolean.